crates/holochain_sqlite/src/schema.rs

```diff
@@ -29,7 +29,13 @@ pub static SCHEMA_CELL: Lazy<Schema> = Lazy::new(|| Schema {
 });
 
 pub static SCHEMA_CONDUCTOR: Lazy<Schema> = Lazy::new(|| Schema {
-    migrations: vec![M::initial(include_str!("sql/conductor/schema/0.sql"))],
+    migrations: vec![
+        M::initial(include_str!("sql/conductor/schema/0.sql")),
+        M {
+            forward: include_str!("sql/conductor/schema/1.sql").into(),
+            _schema: "".into(),
```

Comment:
We need to have a good dev script for printing out the current schema for each db, if we're not gonna have it committed to the repo.